ticdc/ticdc-debezium.md

```diff
@@ -172,4 +786,18 @@ The data format mapping in the TiCDC Debezium message basically follows the [Deb
 
 - For string-like data types, including Varchar, String, VarString, TinyBlob, MediumBlob, BLOB, and LongBlob, when the column has the BINARY flag, TiCDC encodes it as a String type after encoding it in Base64; when the column does not have the BINARY flag, TiCDC encodes it directly as a String type. The native Debezium Connector encodes it in different ways according to `binary.handling.mode`.
 
-- For the Decimal data type, including `DECIMAL` and `NUMERIC`, TiCDC uses the float64 type to represent it. The native Debezium Connector encodes it in float32 or float64 according to the different precision of the data type.
+- For the Decimal data type, including DECIMAL and NUMERIC, TiCDC uses the float64 type to represent it. The native Debezium Connector encodes it in float32 or float64 according to the different precision of the data type.
+
+- TiCDC converts REAL to DOUBLE, and converts BOOLEAN to TINYINT(1) when the length is one.
+
+- In TiCDC, the BLOB, TEXT, GEOMETRY, or JSON column does not have a default value.
+
+- Debezium FLOAT data convert `"5.61"` to `"5.610000133514404"`, but TiCDC does not.
+
+- TiCDC print the wrong `flen` with the FLOAT [tidb#57060](https://github.com/pingcap/tidb/issues/57060).
+
+- Debezium converts `charsetName` to `"utf8mb4"` when the column collation is `"utf8_unicode_ci"` and the character set is null, but TiCDC does not.
+
+- Debezium escapes ENUM elements, but TiCDC does not. For example, Debezium encodes ENUM elements ('c', 'd', 'g,''h') to ('c','d','g,\'\'h').
```

Comment:
Please update the example @wk989898. It does not display well. 
<img width="977" alt="image" src="https://github.com/user-attachments/assets/9b5d1020-8835-4d3e-8d70-d8edffaf896d" />